### PR TITLE
Add classname to that can override text input filtering

### PR DESCRIPTION
**Pull request by ben-chin**
- [x] CR @jeffchan
- [x] QA @ben-chin

### CODE REVIEW

**Comment by jeffchan** [comment]
:+1:

**Comment by jeffchan** [comment]
Can you update test cases?

**Comment by ben-chin** [comment]
cc @jeffchan: e0bb692 and 3563db1

File: test/keymaster.html [hunk 1]
```diff
@@ -399,7 +399,26 @@
       },
 
       testDoesntFireOnUserInputElements: function(t){
-        // TODO
```
Comment (jeffchan) [inline test/keymaster.html]: Lol

**Pull request merged**